src/applications/gi/components/heading/Breadcrumbs.jsx

```diff
@@ -1,42 +1,56 @@
 import { Link, browserHistory } from 'react-router';
 import PropTypes from 'prop-types';
 import React from 'react';
+import Breadcrumbs from '@department-of-veterans-affairs/formation/Breadcrumbs';
 
-class Breadcrumbs extends React.Component {
+class GibBreadcrumbs extends React.Component {
   render() {
-    const { pathname, query: { version } } = this.props.location;
+    const {
+      pathname,
+      query: { version },
+      search
+    } = this.props.location;
+    const facilityCode = this.props.facilityCode;
+    const root = { pathname: '/', query: (version ? { version } : {}) };
 
     const crumbs = [
       <a href="/" key="home">Home</a>,
       <a href="/education/" key="education">Education Benefits</a>,
       <a href="/education/gi-bill/" key="gi-bill">GI Bill</a>,
+      <Link to={root} key="main">GI Bill® Comparison Tool</Link>
     ];
 
-    if (pathname.match(/search|profile/)) {
-      const root = { pathname: '/', query: (version ? { version } : {}) };
-      crumbs.push(<Link to={root} key="main">GI Bill® Comparison Tool</Link>);
+    if (pathname.match(/search/)) {
+      crumbs.push(<Link to={`/search/${search}`} key="search-results">Search Results</Link>);
     }
 
     if (pathname.match(/profile/)) {
       if (this.props.includeSearch) {
-        crumbs.push(<a onClick={browserHistory.goBack} key="search-results">Search Results</a>);
+        crumbs.push(
+          <a
+            onClick={browserHistory.goBack}
+            key="search-results">Search Results
+          </a>,
+          <Link to={`/profile/${facilityCode}`} key="result-detail">School Details</Link>
+        );
+      } else {
+        // User copy-pastes a profile URL, or refreshes page
+        crumbs.push(
```

Comment:
Could we move this `crumbs.push` statement beneath `if (pathname.match(/profile/)) {` so we can avoid duplicating it?